bundles/Billing/Resources/assets/pages/HomePage.js

```diff
@@ -251,6 +251,9 @@ class HomePage extends Component {
                                         <th>{t('home_page.table.project')}</th>
                                         <th>{t('home_page.table.date')}</th>
                                         <th>{t('home_page.table.amount')}</th>
+                                        {tab.keyEvent === 'posted' &&
+                                            <th>{t('home_page.table.exported_date')}</th>
```

Comment:
Some will be confused by this clever use of `&&`. Is this a recommended use in React?

Reply:
https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator